gmail/src/utils/http.ts

```diff
@@ -3,7 +3,7 @@ import { State } from "../models/state";
 /**
  * Make a JSON RPC call with the following parameters.
  */
-export function postJsonRpc(url: string, data = {}, headers = {}) {
+export function postJsonRpc(url: string, data = {}, headers = {}, returnRawResponse: boolean = false) {
```

Comment:
Hmmm how about a "options" dict instead?
That way when calling the method the caller specifies "{returnRawResponse: true}" in the params, which is a bit more clear than a random true appearing at the end of the other parameters.

Reply:
Good idea